--- lib/mpl_toolkits/basemap/__init__.py    2017-05-04 20:53:42.000000000 +0000
+++ lib/mpl_toolkits/basemap/__init__.py    2018-04-27 12:26:14.113683526 +0000
@@ -4780,7 +4780,7 @@
             else:
                 itemindex = 0

-            if fix_wrap_around and itemindex:
+            if False:
                 # check to see if cyclic (wraparound) point included
                 # if so, remove it.
                 if np.abs(lonsin[0]-lonsin[-1]) < 1.e-4:
